packages/fxa-support-panel/package.json

```diff
@@ -32,6 +32,7 @@
   "author": "Mozilla (https://mozilla.org/)",
   "readmeFilename": "README.md",
   "dependencies": {
+    "@hapi/joi": "15.1.1",
```

Comment:
Q: "@hapi/joi" (v15) or "joi" (v17)?

Reply:
@pdehaan, joi is not used in fxa-support-panel, but the typesafe-joi dependency is a fork of hapi/joi (15.x). Adding hapi/joi v15 eliminates the following message from appearing after `yarn install`:
`➤ YN0002: │ fxa-support-panel@workspace:packages/fxa-support-panel doesn't provide @hapi/joi (pf2118), requested by typesafe-joi`